examples/jsm/nodes/accessors/Object3DNode.js

```diff
@@ -16,7 +16,7 @@ class Object3DNode extends Node {
 
 		this.updateType = NodeUpdateType.OBJECT;
 
-		this._uniformNode = uniform( null );
+		this._uniformNode = new UniformNode( null );
```

Comment:
What is the difference here?

Reply:
It was part of what I described about avoiding NodeObject ( `Use Non-NodeObject for uniforms.` ) that need to be updated for every frame.

Reply:
But what is the benefit then? `UniformNode` is still a node.

Reply:
> But what is the benefit then?

Better performance.

Reply:
Is there a significant difference in performance between a node and a node wrapped in `nodeObject`?